concepticondata/conceptlists/KBA-2021-912.tsv

```diff
@@ -0,0 +1,913 @@
+KBA-2021-912-802	802	m629-k-bp-fl-fa	to foresee (predict, divine)				présager, prédire	vorhersagen, weissagen, prophezeien	voorspel	go ramasedi		9. The Supernatural	M569		
+KBA-2021-912-803	803	m63-k-bp-fl-fa	to smash	2558	BREAK (CLEAVE)		fracasser, briser (en morceaux)	zerschmettern, zertrümmern, kaputtschlagen	verbrysel, stukkend slaan/gooi	go kgabetlela		1C. Social and Antisocial Behaviour	M237		
+KBA-2021-912-804	804	m630-k-bp-fl-fa	herbalist					Naturheilkundige, Naturheilpraktiker	kruiedokter	ngaka ya setso		9. The Supernatural	M570		
+KBA-2021-912-805	805	m631-k-bp-fl-fa	sorcerer, witch				sorcier,sorcière	Zauberer, Zauberin; Hexe	towenaar	moloi		9. The Supernatural	M571		
+KBA-2021-912-806	806	m632-k-bp-fl-fa	to bewitch	118	BEWITCH			verzaubern, verhexen	betower	go lowa		9. The Supernatural	M572		
+KBA-2021-912-807	807	m633-k-bp-fl-fa	witchcraft				sorcerie	Zauberei, Hexerei	toorkuns	boloi		9. The Supernatural	M573		
+KBA-2021-912-808	808	m634-k-bp-fl-fa	big (large)	1202	BIG		grand, gros	groß	groot	botona		10. Qualities	M574	s011	lj032
+KBA-2021-912-809	809	m635-k-bp-fl-fa	small	1246	SMALL		petit	klein	klein	bonnye		10. Qualities	M575	s140	lj091
+KBA-2021-912-810	810	m636-k-bp-fl-fa	long	1203	LONG		long	lang	lang, lank	botelele		10. Qualities	M576	s092	lj078
+KBA-2021-912-811	811	m637-k-bp-fl-fa	short	1645	SHORT		court	kurz	kort	bokhutshwane		10. Qualities	M577	s134	
+KBA-2021-912-812	812	m638-k-bp-fl-fa	wide	1243	WIDE		large	breit, weit	wyd	go phatlalala		10. Qualities	M578	s189	lj096
+KBA-2021-912-813	813	m639-k-bp-fl-fa	narrow	1267	NARROW		étroit	eng	eng	go pitlagane		10. Qualities	M579		
+KBA-2021-912-814	814	m64-k-bp-fl-fa	to slap (sb.'s face)	3160	SLAP		donner une claque	ohrfeigen, eine Ohrfeige geben, klatschen	klap gee	go itaya ka mpama		1C. Social and Antisocial Behaviour	M162		
+KBA-2021-912-815	815	m640-k-bp-fl-fa	thin	2308	THIN		mince	dünn	dun	bosesane		10. Qualities		s165	
+KBA-2021-912-816	816	m641-k-bp-fl-fa	thick	1244	THICK		épais,grossier	dick	dik	bokima		10. Qualities		s164	lj076
+KBA-2021-912-817	817	m642-k-bp-fl-fa	lean (thin, skinny)	1400	THIN (SLIM)		maigre	mager	maer			10. Qualities			
+KBA-2021-912-818	818	m643-k-bp-fl-fa	fat	1279	FAT (OBESE)		gros, corpulent	dick, fett	dik, fet	bokima		10. Qualities	M593		
+KBA-2021-912-819	819	m644-k-bp-fl-fa	heavy	1210	HEAVY		lourd	schwer	swaar	bokima		10. Qualities	M580	s071	lj071
+KBA-2021-912-820	820	m645-k-bp-fl-fa	light (in weight)	1052	LIGHT (WEIGHT)		léger	leicht	lig	motlhofo		10. Qualities	M581		
+KBA-2021-912-821	821	m646-k-bp-fl-fa	strong	785	STRONG		fort,puissant	stark, kräftig	sterk			10. Qualities			
+KBA-2021-912-822	822	m647-k-bp-fl-fa	weak	1601	WEAK		faible	schwach	swak	bokoa		10. Qualities	M595		
+KBA-2021-912-823	823	m648-k-bp-fl-fa	hard	1884	HARD		dur	hart	hard	bothata		10. Qualities	M586		lj099
+KBA-2021-912-824	824	m649-k-bp-fl-fa	soft (cloths)	1856	SOFT		doux	weich	saf (kleding)	boleta		10. Qualities	M587		
```

Comment:
This seems to refer to a certain kind of soft, i.e., after washing your clothes with Perwoll. I'd suggest to unmap.

Reply:
Given the translations, I *think* the `(cloths)` is only to disambiguate the gloss, not necessarily to specify it.